packages/services/api/src/modules/schema/providers/schema-publisher.ts

```diff
@@ -532,9 +532,9 @@ export class SchemaPublisher {
         selector,
       });
 
-    const schemaVersionContracts = comparedSchemaVersion
+    const schemaVersionContracts = latestSchemaVersion
```

Comment:
Lets rename this to latestSchemaVersionContracts, to be more explicit

Reply:
did that without even reading your comment :D